README.md

```diff
@@ -23,21 +23,15 @@ We have a Matrix Room Space `#obliteration:matrix.org` on [Matrix.to](https://ma
 
 ## Features
 
-- [x] Built-in PUP file supports for decrypted PUP from [pup_decrypt](https://github.com/idc/ps4-pup_decrypt).
+- [x] Built-in FTP client to pull the decrypted firmware from jailbroken PS4.
 - [x] Built-in PKG file supports for Fake PKG.
 - [x] Game library.
 - [ ] Emulate system calls instead of user-space libraries.
 
 ## System requirements
```

Comment:
Add somewhere in the requirements that a jailbroken ps4 is necessary.

Reply:
should include that an ftp server should be installed as well.

Reply:
Just added it to the system requirements section.